config.toml

```
@@ -1,3 +1,131 @@
 baseURL = "https://unicef.github.io/drone-dpgtoolkit"
 languageCode = "en-us"
 title = "Drone DPG Toolkit"
+theme = "inventory"
+enableEmoji = true
+enableGitInfo = true
+
+# Google Analytics
+googleAnalytics = "G-80FMX4TPM0"
+
+# Allow unsafe HTML
+[markup.goldmark.renderer]
+unsafe = true
+
+
+############################# Navigation #############################
+[[menu.main]]
+weight = 10
+name = "FAQ"
+url = "faq"
+
+[[menu.main]]
+weight = 20
+name = "pages"
+url = "pages"
+hasChildren = true
+
+
+
+
+############################# Default parameters #############################
+[params]
+# Need to replace with SVG logo
+#logo = "favicon.ico"
+
+# OpenGraph / Twitter Card metadata
+description = "UNICEF Open Source Inventory. A UNICEF Global Innovation knowledge base of best practices and resources for working and leading Open."
+author = "Justin W. Flory, UNICEF Office of Innovation, and others"
```

Let's attribute the content correctly:

```suggestion
author = "Edmond Nonie, UNICEF Office of Innovation, and others"
```